app/views/users/index.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Users#id</h1>
-<p>Find me in app/views/users/id.html.erb</p>
+<h1>Users</h1>
+
```

Comment:
- [ ] OPTIONAL
Your design does not match the given wireframe template. Kindly endeavor to improve upon the design.

Your **design:**
![image](https://user-images.githubusercontent.com/12745474/191626794-70d8bed1-b423-4bad-b116-12bfe842faa1.png)

![image](https://user-images.githubusercontent.com/12745474/191626982-39764c24-2e77-48c3-9e52-1cdb01af0894.png)


**The original wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/191626893-9098aecc-9fc1-4774-b0fd-17ef4d63aa41.png)

Reply:
Done